docs/start_nominating.md

```diff
@@ -123,12 +123,34 @@ Once a nomination becomes active, you should find it in the list of *Active nomi
 
 <div style={{textAlign: 'center'}}>
   <img src={useBaseUrl('/nominator-guide/nominate-validator-4.png')} />
-</div>
+</div>  
 
 :::note
 
 Make sure that you revisit your nominations once in a while. It is possible that some of your validators change their commission percentage which will have a negative impact on your rewards. By checking the status of your nominations frequently you will be able to react by updating the list of your nominated validators.
 
 :::
 
+## 04 Change Nominees
```

Comment:
Adjust your nominations

(let's not use Camel Case to remain consistent with other titles)